ui/src/app/views/admin-user-bypass/component.tsx

```diff
@@ -21,114 +21,88 @@ const styles = reactStyles({
   }
 });
 
-export interface AccessModules {
-  complianceTraining: boolean;
-  betaAccess: boolean;
-  eraCommons: boolean;
-  twoFactorAuth: boolean;
-  dataUseAgreement: boolean;
-}
-
 export class AdminUserBypass extends React.Component<
     { profile: Profile},
-    { modules: AccessModules, editedModules: AccessModules,
+    { initialModules: AccessModule[], selectedModules: AccessModule[],
```

Comment:
opt: A set (rather than an array) feels like a better data structure for this task

Reply:
Seems reasonable to me, will update.

Reply:
Talked offline, javascript sets don't support xor. Moving forward with array for now.